content/actions/using-workflows/workflow-syntax-for-github-actions.md

```diff
@@ -157,6 +157,34 @@ jobs:
 ```
 {% endraw %}
 
+{% if actions-inherit-secrets-reusable-workflows %}
+
+#### `on.workflow_call.secrets.inherit`
+
+Use the `inherit` keyword to pass all secrets the calling workflow has access to, to the called workflow. This includes all organization, repository and environment secrets. The keyword can be used to pass secrets across organizations within the same enterprise. If there are duplicate names between the repository and organization, the repository secret will be used.
+
+#### Example
+
+{% raw %}
+
+```yaml
+on:
+  workflow_call:
+    secrets: inherit
+
```

Comment:
We do not have keyword `inherit` in the called workflow. The keyword will be used in the calling workflow

Reply:
so the example should present caller workflow, for example

```yaml
jobs: 
  calling-job:
    uses: ./.github/workflows/called-workflow.yml
    secrets: inherit
```